microapps/MapViewCalloutApp/app/src/main/java/com/arcgismaps/toolkit/mapviewcalloutapp/screens/TapLocationScreen.kt

```diff
@@ -75,32 +74,30 @@ fun TapLocationScreen(viewModel: MapViewModel) {
             }
         )
 
-        Box {
-            MapView(
-                modifier = Modifier
-                    .fillMaxSize(),
-                arcGISMap = viewModel.arcGISMap,
-                graphicsOverlays = remember { listOf(viewModel.tapLocationGraphicsOverlay) },
-                onSingleTapConfirmed = viewModel::setMapPoint,
-                onLongPress = { viewModel.clearMapPoint() },
-                content = if (mapPoint != null && calloutVisibility) {
+        MapView(
+            modifier = Modifier
+                .fillMaxSize(),
+            arcGISMap = viewModel.arcGISMap,
+            graphicsOverlays = remember { listOf(viewModel.tapLocationGraphicsOverlay) },
+            onSingleTapConfirmed = viewModel::setMapPoint,
+            onLongPress = { viewModel.clearMapPoint() },
+            content = if (mapPoint != null && calloutVisibility) {
+                {
+                    Callout(
+                        modifier = Modifier.size(175.dp, 75.dp),
+                        location = mapPoint,
+                        rotateOffsetWithGeoView = rotateOffsetWithGeoView,
+                        offset = offset
+                    )
                     {
```

Comment:
formatting seems off here.

```suggestion
                    ) {
```